src/jit/compiler.h

```diff
@@ -3027,7 +3072,7 @@ protected :
     void                impLoadLoc(unsigned ilLclNum, IL_OFFSET offset);
     bool                impReturnInstruction(BasicBlock *block, int prefixFlags, OPCODE &opcode);
 
-#if defined(_TARGET_ARM_)
+#if defined(FEATURE_HFA)
```

Comment:
This method doesn't need to be inside #if FEATURE_HFA because it is implemented to use IsHfa(hClass). Once the above suggested change is made to IsHfa(), this routine too doesn't need to under FEATURE_HFA.